service/apiserver/server.api

```diff
@@ -451,13 +446,13 @@ service server-api {
 	@handler GetTx
 	get /api/v1/tx (ReqGetTx) returns (EnrichedTx)
 	
-	@doc "Get mempool transactions"
```

Comment:
also need to update the document `api-reference.md`.

Reply:
Yeah, I think so, I have updated the api-reference.md.

Reply:
sorry, missed it